pkg/virt-handler/device-manager/device_controller.go

```diff
@@ -276,19 +333,21 @@ func (c *DeviceController) Run(stop chan struct{}) error {
 	<-stop
 
 	// stop all device plugins
-	c.devicePluginsMutex.Lock()
-	for _, dev := range c.devicePlugins {
-		dev.stopChan <- struct{}{}
-	}
-	c.devicePluginsMutex.Unlock()
+	func() {
+		c.startedPluginsMutex.Lock()
+		defer c.startedPluginsMutex.Unlock()
+		for name, _ := range c.startedPlugins {
```

Comment:
Since it is not being used, you can omit the second returned value from the range:

```golang
for name := range c.startedPlugins {
...
}
```